index.rst

```diff
@@ -528,6 +528,13 @@ For all routes listed below with a ``username`` path parameter, only administrat
     Only user tokens may be created this way.
     Tokens of other types are created through non-API flows described later.
     The token name, scopes, and desired expiration are provided as parameters.
+    The newly-created token is returned as follows:
+
+    .. code-block:: json
+
+       {
+         "token": "gt-qVGZIh65TAJlNprOaMDhwg.WlUA5zyAY16dDRvDYxnwhg"
```

Comment:
Is it `gt` or `gsh`?

Reply:
`gt-`, sorry. Fixed in the next version.